manifests/charts/istio-control/istio-discovery/templates/mutatingwebhook.yaml

```diff
@@ -4,6 +4,7 @@
  "revision" .Values.revision
   "injectionPath" .Values.istiodRemote.injectionPath
   "injectionURL" .Values.istiodRemote.injectionURL
+  "reinvocationPolicy" .Values.sidecarInjectorWebhook.reinvocationPolicy
```

Comment:
I think this is not the only place to add the field, and perhaps this needs to be added in the operator spec as well.